app/models/network_port.rb

```diff
@@ -0,0 +1,17 @@
+class NetworkPort < ActiveRecord::Base
+  include NewWithTypeStiMixin
+  include ReportableMixin
+
+  belongs_to :ext_management_system, :foreign_key => :ems_id, :class_name => "ManageIQ::Providers::CloudManager"
+  belongs_to :cloud_tenant
+  belongs_to :cloud_network
+  belongs_to :cloud_subnets
```

Comment:
Still trying to get this straight in my head.

So, it sounds like "cloudy" network objects (`cloud_network` and `cloud_subnet`) are going to have to have some "infra" representation because OpenStack Infra also uses Neutron.

But, I think at the very least, the relationships `network_router => cloud_tenant` and `network_port => cloud_tenant` could at least be moved to the cloud-specific models, right?

In fact, maybe that means creating `network_router` and `network_port` at the [base cloud manager](https://github.com/Ladas/manageiq/tree/openstack_missing_neutron_models/app/models/manageiq/providers/cloud_manager) level and introduce those relationships there.  That way, only cloudy `network_routers` and `network_ports` have access to the list `cloud_tenant`.  Unless the infra layer will also have a `cloud_tenant` (which I really hope it doesn't).